cloudapi/api.go

```diff
@@ -258,3 +265,27 @@ func (c *Client) Login(email string, password string) (*LoginResponse, error) {
 
 	return &lr, nil
 }
+
+// ValidateToken calls the endpoint to validate the Client's token and returns the result.
+func (c *Client) ValidateToken() (*ValidateTokenResponse, error) {
+	url := fmt.Sprintf("%s/validate-token", c.baseURL)
+
+	data := struct {
+		Token string `json:"token"`
+	}{
+		c.token,
+	}
```

Comment:
Maybe better if this is named struct `validateTokenRequest`

Reply:
Agreed 👍